build.gradle

```diff
@@ -142,7 +142,8 @@ task run(type: JavaExec) {
         println "Using LOG_DIR: $logDirectory"
         System.setProperty "LOG_DIR", "$logDirectory"
 
-        def configFile = System.getenv("CONFIG")
+//        def configFile = System.getenv("CONFIG")
```

Comment:
remove commented code

Reply:
don't change anything in build.gradle, don't remove this line as it is fetching the path for config from command line while executing. Uncomment it instead.

Reply:
remove this line def configFile = System.getProperty("user.dir") + "/configs/vodqa_local_config.properties";